ros/processor/event_producer.py

```diff
@@ -0,0 +1,76 @@
+import json
+from flask import request
+from ros.lib import produce
+from confluent_kafka import KafkaError
+from datetime import datetime, timezone
+from ros.lib.models import PerformanceProfile
+from ros.lib.config import NOTIFICATIONS_TOPIC, get_logger
+from ros.lib.utils import org_id_from_identity_header, systems_ids_for_existing_profiles
+
+logger = get_logger(__name__)
+
+
+# Event for new suggestion
+def new_suggestion_event(host, platform_metadata, previous_state, current_state):
+
+    org_id = org_id_from_identity_header(request)
```

Comment:
IIUC for existing processors, we are fetching org_id from either host object or platform_metadata.
I didn't get this part. Is request object present for background processing?
Why can't you use `host` object here to fetch `org_id` same way you did here -> [line](https://github.com/RedHatInsights/ros-backend/pull/257/files#diff-7f50c9ee82de30d0024864fef8fd7f6b8245e30a4ae9058582844eb544a52156R27)